x86_64-unknown-intermezzos-gnu.json

```diff
@@ -3,15 +3,20 @@
 	"cpu": "x86-64",
 	"data-layout": "e-m:e-i64:64-f80:128-n8:16:32:64-S128",
 	"executables": true,
-        "linker-flavor": "gcc",
+	"linker": "gcc",
```

Comment:
1. I don't know which one takes precedence, the cargo config or the target spec.
2. I'm also not sure how the string is looked up to a full binary path, I expect this to be a simple search in PATH. I did some digging and am temporarily stuck here: https://github.com/rust-lang/rust/blob/5dfcd85fd4bae49445383baadf472fbdb414a0e6/src/librustc_trans/back/link.rs#L149-L159